README.md

```diff
@@ -1,64 +1,29 @@
-# stylelint-config-kraftvaerk 
-Stylelint shareable config for the Kraftvaerk stylesheet style guide
+# stylelint-config-kraftvaerk
 
-[![GitHub release](https://img.shields.io/github/release/kraftvaerk/stylelint-config-kraftvaerk.svg?style=flat-square)]() [![Build Status](https://img.shields.io/travis/kraftvaerk/stylelint-config-kraftvaerk/master.svg?style=flat-square)](https://travis-ci.org/kraftvaerk/stylelint-config-kraftvaerk) [![David](https://img.shields.io/david/dev/kraftvaerk/stylelint-config-kraftvaerk.svg?style=flat-square)]()
+[![Build Status](https://img.shields.io/travis/kraftvaerk/stylelint-config-kraftvaerk/master.svg?style=flat-square)](https://travis-ci.org/kraftvaerk/stylelint-config-kraftvaerk) [![npm Version](https://img.shields.io/npm/v/stylelint-config-kraftvaerk.svg?style=flat-square)](https://www.npmjs.com/package/stylelint-config-kraftvaerk) [![npm Downloads](https://img.shields.io/npm/dm/stylelint-config-kraftvaerk.svg?style=flat-square)](https://www.npmjs.com/package/stylelint-config-kraftvaerk) [![devDependency Status](https://img.shields.io/david/dev/kraftvaerk/stylelint-config-kraftvaerk.svg?style=flat-square)](https://david-dm.org/kraftvaerk/stylelint-config-kraftvaerk/?type=dev)
 
-[![npm](https://nodei.co/npm/stylelint-config-kraftvaerk.svg?downloads=true)](https://nodei.co/npm/stylelint-config-kraftvaerk/)
+> Kraftvaerk's shareable config for stylelint
 
+## Rules for your CSS
 
-# Rules for your CSS
-
-Code should always be written in such a way that makes it look like a single person wrote it, no matter how many people have contributed and that is the purpose of this set of rules powered by [stylelint](https://github.com/stylelint/stylelint).
+Code should always be written in such a way that makes it look like a single person wrote it, no matter how many people have contributed and that is the purpose of this set of rules powered by [stylelint](https://stylelint.io/).
 
 ## Installation
 
-You need to have `stylelint` and this config as part of your `devDependencies`.
-
 ```
-$ npm install --save-dev stylelint stylelint-config-kraftvaerk
+npm install --save-dev stylelint stylelint-config-kraftvaerk
 ```
 
 ## Usage
 
-### Load everything
-
-To use all the rules all at once add `stylelint-config-kraftvaerk` to your `extends` property in `.stylelintrc`.
+To use it set your `extends` property in `.stylelintrc` to `stylelint-config-kraftvaerk`.
 
 ```json
 {
-  "extends": [
-    "stylelint-config-kraftvaerk"
-  ]
+  "extends": "stylelint-config-kraftvaerk"
 }
 ```
 
-### Load a specific rule
-
-Each rule for `stylelint-config-kraftvaerk` can be loaded individually if you only want to load a specific rule. They are all located in the `lib` folder.
-
-Load the `color` and `general` rule set.
+## License
 
-```json
-{
-  "extends": [
-    "stylelint-config-kraftvaerk/lib/color",
-    "stylelint-config-kraftvaerk/lib/general"
-  ]
-}
-```
-
-
-### Override specific rules
```

Comment:
I think actual information on this topic could always be found on the stylelint's website, so we don't have to care whether they've changed API or something. Does it make sense? Please let me know if we need to have it in our `README.md`.

Reply:
I totally agree